frontend/ui/elements/reader_menu_order.lua

```diff
@@ -202,6 +202,7 @@ local order = {
         "wikipedia_history",
         "wikipedia_settings",
         "----------------------------",
+        "translate_page",
```

Comment:
This will add a 11th item in that menu (10th for me with Vocabulary builder disabled), so making it 2 pages :/
No real better idea. In the 2nd menu with a page icon? so-so.
May be the dict & wiki lookup history should be moved into their respecting ... settings, or be in the own submenu?

Reply:
12 (incl. calibre search).
Maybe move dict, wiki and translation settings into Settings submenu?